playbooks/roles/ocp-upgrade/tasks/eus_upgrades.yaml

```diff
@@ -1,5 +1,10 @@
 ---
 
+- name: Get intial kubelet version
+  kubernetes.core.k8s_info:
+    kind: Node
+  register: initial_kubeletversion
```

Comment:
Can we change the name to `node_info` instead of kubeletversion?

Reply:
I will change the name